crates/starknet-types-core/src/curve/affine_point.rs

```diff
@@ -0,0 +1,45 @@
+use crate::curve::curve_errors::CurveError;
+use crate::felt::Felt;
+use lambdaworks_math::cyclic_group::IsGroup;
+use lambdaworks_math::elliptic_curve::short_weierstrass::curves::stark_curve::StarkCurve;
+use lambdaworks_math::elliptic_curve::short_weierstrass::point::ShortWeierstrassProjectivePoint;
+use lambdaworks_math::elliptic_curve::traits::FromAffine;
+
+#[derive(Debug, Clone, PartialEq, Eq)]
+pub struct AffinePoint(pub(crate) ShortWeierstrassProjectivePoint<StarkCurve>);
```

Comment:
Can you add a short description of this type plz

Reply:
Done!
[3a20aed](https://github.com/starknet-io/types-rs/pull/8/commits/3a20aed7cfed518017454a2b24c393c101cb3189)